.circleci/config.yml

```diff
@@ -60,6 +73,37 @@ jobs:
           command: |
             tox -e stylechecks
 
+  package_and_upload_to_pypi:
+    <<: *defaults
+    steps:
+      - checkout
+      - *prepare_venv
+      - run:
+          name: Install requirements
+          command: |
+            . venv/bin/activate
+            pip install -r requirements.txt
+            pip install -e .
```

Comment:
not sure I need to install reqs if I am installing from the setup below? this was confusing in the file from which I took inspiration. The reqs are part of the setup.py, so shouldb't be needed as additional line. right? @ChristopherGS